File: 2 Array/Basics/uniqueOccurences.py
def func():
    arr = [-3,0,1,-3,1,1,1,-3,10,0]
    arr.sort()
    print(arr)
    arr2=[]
    for i in arr:
        count = 0
        for j in arr:
            if i==j:
                count+=1
        if i not in arr2:
            arr2.append(i)
    occ=[]
    for i in arr2:
        count=0
        for j in arr:
            if i==j:
                count+=1
        occ.append(count)
            
    print(arr2)
    print(occ)
    
    for i in range(0,len(occ)):
        for j in occ[i:]:
            if i==j:
                return False
                   
    return True
    
  
        
    
print(func())

File: 2 Array/Basics/_uniqueOccurences.cpp
#include <iostream>
using namespace std;

void uniqueOcc(int arr[],int size){
    int countArr[size] ;
    for(int i = 0; i < size; i++){
        int count = 1;
        for(int j=i+1; j<size ;j++){
            if(arr[i]==arr[j]){
                count++;
            }
        }
        countArr[i];

    } 
}

int main(){
    int arr[6]={2,2,3,3,3,1};
    int size = sizeof(arr)/sizeof(int);
    cout<<size;
    uniqueOcc(arr,size);
    // arr.size() -> vector ka size


}